store/mongo/datastore_mongo.go

```diff
@@ -288,15 +288,16 @@ func NewMongoClient(ctx context.Context, c config.Reader) (*mongo.Client, error)
 	wc.WithOptions(writeconcern.W(1), writeconcern.J(true))
 	clientOptions.SetWriteConcern(wc)
 
```

Comment:
changes in this file shouldn't be a part of "vendor" commit

Reply:
Well spotted! I'll move it back to where it belongs.